conf/cassandra.yaml

```diff
@@ -2368,3 +2368,104 @@ drop_compact_storage_enabled: false
+#      # Let's say there is a Cassandra cluster in that there are 10 tables belonging to 10 different customers.
+#      # Out of these 10 tables, 1 table is humongous. Repairing this 1 table, say, takes 5 days, in the worst case, but others could finish in just 1 hour.
+#      # Then we would penalize 9 customers just because of one bad actor, and those 9 customers would ping an operator and would require a lot of back-and-forth manual interventions, etc.
+#      # So, the idea here is to penalize the outliers instead of good candidates. This can easily be configured with a higher value if we want to disable the functionality.
+#      table_max_repair_time: 6h
+#      # This is useful if you want to completely avoid running repairs in one or more data centers. By default, it is empty, i.e., the framework will repair nodes in all the datacenters.
+#      # If you want to avoid repairing nodes in one or more data centers, then you can specify the data centers in this list.
+#      ignore_dcs: []
+#      # Set this 'true' if AutoRepair should repair only the primary ranges owned by this node; else, 'false'.
+#      # It is the same as -pr in nodetool repair options.
+#      repair_primary_token_range_only: true
+#      # The number of nodes running repair parallelly. If parallel_repair_percentage is set, it will choose the larger value of the two.
+#      # This configuration controls how many nodes would run repair in parallel.
+#      # The value “3” means, at any given point in time, at most 3 nodes would be running repair in parallel. These selected nodes can be from any datacenters.
+#      # If one or more node(s) finish repair, then the framework automatically picks up the next candidate(s) based on the least repair time.
+#      # It will ensure the maximum number of nodes running repair do not exceed “3”.
+#      parallel_repair_count: 3
+#      # The percentage of nodes in the cluster that run repair parallelly. If parallel_repair_count is set, it will choose the larger value of the two.
+#      # The problem with a fixed number of nodes (parallel_repair_count property) is that in a large-scale environment,
+#      # the nodes keep getting added/removed due to elasticity, so if we have a fixed number, then manual interventions would increase because, on a continuous basis,operators would have to adjust to meet the SLA.
+#      # The default is 3%, which means that 3% of the nodes in the Cassandra cluster would be repaired in parallel.
+#      # So now, if a fleet, an operator won't have to worry about changing the repair frequency, etc., as overall repair time will continue to remain the same even if nodes are added or removed due to elasticity.
+#      # Extremely fewer manual interventions as it will rarely violate the repair SLA for customers.
+#      parallel_repair_percentage: 3
+#      # If the scheduler should repair MV table or not.
+#      mv_repair_enabled: false
```

Comment:
Just realized, we don't use `mv` prefix for any other properties in `cassandra.yaml` so maybe we should expand this to `materialized_view_repair_enabled`? wdyt?

Reply:
renamed